client/app/lib/components/redux-form/Checkbox.js

```diff
@@ -5,15 +5,15 @@ export default createComponent(
   Checkbox,
   ({
     input: { onChange, value, ...inputProps },
-    meta, // eslint-disable-line no-unused-vars
-    intl, // eslint-disable-line no-unused-vars
+    meta: _meta,
```

Comment:
just wondering why do we add _ here?

Reply:
Ah cos it's unused. It's more of a convention, and you can configure the linting to specifically ignore such variables.